views/index.hbs

```diff
@@ -7,6 +7,7 @@
   <meta id="viewport" name="viewport" content="initial-scale=1.0, maximum-scale=1.0, user-scalable=0"/>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   {{baseCss}}
+  <link rel="stylesheet" href="/css/ie.css" type="text/css" media="all">
```

Comment:
Shouldn’t this be in a if ie comment as well?

Reply:
If IE comments only works up to IE 9 after that the if command is not interpreted anymore